src/main/java/command/ExitCommand.java

```diff
@@ -0,0 +1,50 @@
+package command;
+
+import main.Storage;
+import main.TaskList;
+import main.TessUi;
+import main.TesseractException;
+
+/**
+ * Represent a command to exit Tesseract.
+ * @author Fan Jue
+ * @version 0.1.0
+ * @since 0.1.0
+ */
+public class ExitCommand extends Command {
+    /** The specific command keyword */
```

Comment:
I think you forgot to delete this comment?

Reply:
Yes haha good catch!